src/objects/BatchedMesh.js

```diff
@@ -195,8 +195,7 @@ class BatchedMesh extends Mesh {
 				const { array, itemSize, normalized } = srcAttribute;
 
 				const dstArray = new array.constructor( maxVertexCount * itemSize );
-				const dstAttribute = new srcAttribute.constructor( dstArray, itemSize, normalized );
-				dstAttribute.setUsage( srcAttribute.usage );
```

Comment:
But...isn't honoring `usage` a good thing. Let's say you are using `DynamicDrawUsage` because you frequently want to update attribute data then the original setting is now lost.

Reply:
I think if the user need to do this they can set it on `BatchedMesh.geometry` attributes directly. I'm not sure if this is something that I see needs to be copied from the original geometry but I could be convinced. The update usage of BatchMesh may not match the update usage for the first geometry passed to the function.

Reply:
Okay, we can add it back later, too, if a user reports any issues with this change.